docker-php-scripts/docker-php-source

```diff
@@ -14,7 +14,7 @@ usage() {
 	echo
 }
 
-PHP_VER=$(php -v | awk 'NR==1{print $2}')
+PHP_VER=$(php -v | xargs | awk -F 'PHP ' '{print $2}' | awk '{print $1}')
```

Comment:
```suggestion
PHP_VER=$(php -v | xargs | awk '{print $2}')
```

Reply:
@ryanwohara - wouldn't this still get the second value of the first line, which was the original issue?

now that i'm thinking about it, `php -v | grep '[0-9]\.[0-9]' | xargs | awk '{print $2}'` would probably be better

Reply:
I think you nailed it with that `grep`